slobodan/arrays/package.json

```
@@ -0,0 +1,13 @@
+{
+  "name": "arrays",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node homework.js"
```

Also, we can have multiple scripts. :)

```suggestion
    "test": "echo \"Error: no test specified\" && exit 1",
    "homework": "node homework.js",
    "start": "node index.js"
```

Now you can do `npm start homework`